bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -50,11 +53,22 @@ spec:
           - description: The size of deployment profile for IBM Cloud Pak foundational services
             displayName: Size
             path: size
-            x-descriptors:
```

Comment:
Wondering if we remove the `x-descriptors` for `size`. Would user be able to select one of sizes in creation view instead of input the text?